src/App.js

```diff
@@ -24,20 +26,22 @@ class App extends Component {
   componentDidMount = async() => {
     const data = await fetchData(this.state.query, this.state.direction);
     this.setState({data})
+    this.setState({loading:false})
   }
   
   render() { 
-    const { load } = this.state;
-    const filterType = this.data.filter(
-      (pokemon) => {
-        return (this.data.type_1 === 'All' || this.data.type_1 === pokemon.type_1)
-      }
-    )
-    return ( 
-      <>
+    const {data} = this.state;
+    const {loading} = this.state;
+    const filterType = data.filter(
+      (pokemon) => 
+         pokemon.typeOptionList === this.state.typeOptionList || this.state.typeOptionList === data
```

Comment:
I remember working on this part with you so I'm guessing this was changed in a future PR. I'll keep an eye out to verify.